appveyor.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
As far as I'm aware, ZFS isn't supported on Windows at all, should we not check in the `appveyor.yml`, as I don't think there's much reason to run the tests on Windows?